### ISSUE TITLE
Date being printed the day before 

### ISSUE THREAD

**Issue by starmon00** [SheetJS/sheetjs#1346]
So this issue may be a duplicate or just me not understanding the time zone, but right now the dates (on a csv file) is being read but converted to one day before the actual date. This only happens when the date is in May, June, July, August, September, and October.

My Code:
```
    const studentSheets = xlsx.read(buffer, { type: 'buffer', codepage: 65001 });
    const studentSheet = studentSheets.Sheets[studentSheets.SheetNames[0]];
    logger.info(`studentSheet: ${util.inspect(studentSheet)}`);
```

The Output:
<img width="1292" alt="screen shot 2018-11-07 at 4 40 57 pm" src="https://user-images.githubusercontent.com/7799546/48170334-f1464780-e2ab-11e8-8637-e480b1570e75.png">

Here is the link to the CSV file:
https://docs.google.com/spreadsheets/d/1jKiyzAB_zq_UlOC2LNNNadSGtDJAWEx5elVtbCHI6tc/edit?usp=sharing

**Comment by phani17c** [comment]
@SheetJSDev Please reply to this.

**Comment by phani17c** [comment]
Hey @starmon00, Did you get resolved with the above issue?

**Comment by SheetJSDev** [comment]
This was a DST issue which was fixed when date handling was changed in 548396f87db576bfe2b9d80724554e51211d44f9
